frontend/app/route-helpers/apply-adult-child-route-helpers.server.ts

```diff
@@ -71,15 +71,21 @@ export function loadApplyAdultSingleChildState({ params, request, session }: Loa
   }
 
   const childId = parsedChildId.data;
-  const childState = applyState.children.find(({ id }) => id === childId);
+  const childStateIndex = applyState.children.findIndex(({ id }) => id === childId);
```

Comment:
we should look into this in a future PR.  This won't re-order the child number if, for example, I add 2 children, and delete the first one.  Maybe that's how you want it to be, but I don't know.

Reply:
The `Add a child` action on the children HUB, will add a new child to the list and only keep the children that is now new.

```tsx
const childId = randomUUID();
const children = [...getChildrenState(state, false), { id: childId }];
saveApplyState({ params, session, state: { children } });
```

Reply:
Right, but what I'm getting at in my example is that if I add 2 children, delete the first, then add a new child, the title displayed on the child information page is "Child: 1 information".  Is that what you would expect, or should it be "Child 2: information"

Reply:
With my changes it will be Child 2 since you removed one child.

Reply:
it always says "child 1"?  Anyways, I'll approve this, but I think we'll have to come back to it at some point.